### DISCUSSION TITLE
Heave RAO comparison from regular wave and white noise spectrum.

### DISCUSSION THREAD

**Discussion by darkstar71** [OpenFAST/openfast#2410] [Q&A]
Hi,
For a semi-submersible type of FOWT platform, similar to the DeepC, I ran a few regular wave only simulations in openFAST. The purpose of these simulations were to produce heave RAO values near to the heave natural frequency of the platform. Later, I ran White Noise simulations with the duration of 180 mins for the same wave heights used for regular wave simulations. While comparing the RAO values near the heave natural frequency of the platform the followings are observed:

**Observations:**
1) Heave RAO values from regular wave simulations are significantly lower compared to the White Noise simulations near heave natural period for smaller wave heights, for example, 2 m.
2) As the wave height increases, the difference in heave RAO value from regular wave simulation approached to the same from White Noise simulations but still the differences are around 15-20%.

I am interested to know the possible reasons for these differences? Am I doing anything incorrectly?

Best regards,

**Comment by jjonkman** [comment]
Dear @darkstar71,

Can you clarify how you are determining the RAO from each set up?  For regular waves, are you allowing all start-up transients to die out before measuring the amplitude of the response?  For white noise waves, are you computing the ROAs based on the approach used in the following paper: https://research-hub.nrel.gov/en/publications/investigation-of-response-amplitude-operators-for-floating-offsho-2?

Keep in mind that an RAO assumes linearity, whereas OpenFAST includes nonlinearities, including nonlinearities that can be wave-amplitude dependent, which is why RAOs you compute from OpenFAST using either approach may depend on the wave height.

Best regards,
